src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETIC = 'alphabet',
+  LENGTH = 'length',
+}
+
+const getGoods = (goods: string[], sortType: string, isReversed: boolean) => {
```

Comment:
`sortType` attribute must have the union type `SortType` (that enum) or '' . You should do sorting only if `sortType` is not an empty value. So you need to change your logic a bit